benchmarks/graphics_pipeline/CMakeLists.txt

```diff
@@ -23,4 +23,5 @@ add_samples_for_all_apis(
       "shader_benchmark_vs_alu_bound"
       "shader_benchmark_ps_simple"
       "shader_benchmark_ps_alu_bound"
-      "shader_benchmark_ps_mem_bound")
```

Comment:
`shader_benchmark_ps_unlit` if you want to use the copy you made, or remove the copy if you want to reuse the other unlit shader.

Reply:
Done.